contracts/extensions/VotingReputation.sol

```diff
@@ -605,11 +605,32 @@ contract VotingReputation is ColonyExtension, PatriciaTreeProofs {
 
     if (canExecute) {
       executed = executeCall(_motionId, motion.action);
+      require(executed || failingExecutionAllowed(_motionId), "voting-execution-failed-not-one-week");
     }
 
     emit MotionFinalized(_motionId, motion.action, executed);
   }
 
+
+  /// @notice Return whether a motion, assuming it's in the finalizable state,
+  // is allowed to finalize without the call executing successfully.
+  /// @param _motionId The id of the motion
+  /// @dev We are only expecting this to be called from finalize motion in the contracts.
+  /// It is marked as public only so that the frontend can use it.
+  function failingExecutionAllowed(uint256 _motionId) public returns (bool) {
```

Comment:
Can we mark this as `view`?

Reply:
Oh, yeah, this definitely should be `view` otherwise it's useless for the frontend 😄